samples/src/MSTRWeb/enable-on-certain-dossier/global.js

```diff
@@ -0,0 +1,92 @@
+(function (config) {
+  const PATH_TO_HYPER_JS = '/static/hyper/sdk/js/mstr_hyper.bundle.js';
+  const joinUrl = (baseUrl, apiUrl) =>
+    `${baseUrl.replace(/\/+$/g, '')}/${apiUrl.replace(/^\/+/g, '')}`;
+
+  //your hyper card's project id
+  const ProjectID = 'PROJECT_ID';
+
+  const dossierMap = {
+    //the dossier id you want to show highlight
+    'DOSSIER_ID': [
+      {
+        // the hyper card id you want to enable on this certain dossier
+        id: 'HYPER_CARD_ID',
```

Comment:
rename as "cardID" just to avoid confussion

Reply:
`id` is required by the enable card interface https://microstrategy.github.io/hyper-sdk/code-examples/, we can't change it unless add more code to rename the parameter

Reply:
understood. Lets keep it then.